fxsampler/src/main/java/fxsampler/SampleBase.java

```diff
@@ -167,4 +167,9 @@ public static Node buildSample(Sample sample, Stage stage) {
 
         return splitPane;
     }
+
+    @Override
+    public void dispose() {
+//        System.out.println("Dispose " + getSampleName());

```

Comment:
I guess the commented out system.out should go.

Reply:
I viewed it as sort of a sample  code :)